src/components/gameScreen/gameScreen.container.js

```diff
@@ -33,15 +33,16 @@ export const GameScreenContainer = () => {
   const [isButtonPressed, setIsButtonPressed] = useState(false);
   const translateX = useSharedValue(400)
   useEffect(() => {
-    const filteredQuestion = questions.data.filter((questionItem) => questionItem.text === currentQuestion);
-    const slicedAnswers = Object.values(filteredQuestion[0].answers[0]).slice(3);
-    setCurrentScore(score)
-    setFilteredQuestion(filteredQuestion);
-    setCurrentAnswers(slicedAnswers);
+    const _filteredQuestion = questions.find((questionItem) => questionItem.text === currentQuestion);
```

Comment:
why is it called filtered question when you call *find* on it? You are not filtering it.
Also, find can return undefined, but you are not making any checks for that, this code is not safe